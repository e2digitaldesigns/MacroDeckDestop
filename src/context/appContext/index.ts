/* istanbul ignore file */
export * from "./appContext";
export * from "./defaultState";
