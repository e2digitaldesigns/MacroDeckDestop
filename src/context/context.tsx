import React from "react";
import { TemplateContextProvider } from "./templateContext";
import { ThemeContextProvider } from "./themeContext";

export interface IThemeComponents {
  TemplateProvider: React.ElementType;
  ThemeProvider: React.ElementType;
}

export const ApplicationContext: React.FC & IThemeComponents = () => <div />;

ApplicationContext.TemplateProvider = TemplateContextProvider;
ApplicationContext.ThemeProvider = ThemeContextProvider;
