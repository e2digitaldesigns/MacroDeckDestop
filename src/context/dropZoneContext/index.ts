export * from "./dropZoneContext";
export * from "./defaultState";
