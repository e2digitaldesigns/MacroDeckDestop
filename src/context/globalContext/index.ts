export * from "./globalContext";
export * from "./defaultState";
