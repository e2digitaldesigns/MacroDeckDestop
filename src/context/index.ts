export * from "./appContext";
export * from "./dropZoneContext";
export * from "./globalContext";
export * from "./templateContext";
export * from "./themeContext";
