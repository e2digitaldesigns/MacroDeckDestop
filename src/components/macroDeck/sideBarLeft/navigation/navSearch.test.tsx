import React from "react";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import NavSearch from "./navSearch";

import { ApplicationContext } from "../../../../context/context";

// const testSetup = () => {
//   return render(
//     <ApplicationContext.ThemeProvider>
//       <NavSearch />
//     </ApplicationContext.ThemeProvider>
//   );
// };

// describe("<Left Navigation Search/>", () => {
//   const wrapper = testSetup();
//   xit("Should render without errors", () => {
//     const component = wrapper;
//     expect(component).toMatchSnapshot();
//   });
// });
