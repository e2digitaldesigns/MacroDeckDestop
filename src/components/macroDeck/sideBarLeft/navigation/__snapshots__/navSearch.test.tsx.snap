// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Left Navigation Search/> Should render without errors 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bczRLJ hWayYc"
      >
        <div
          class="sc-gsnTZi hYgUKR"
        >
          <svg
            fill="currentColor"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M8 0a8 8 0 1 0 0 16A8 8 0 0 0 8 0zm3.5 7.5a.5.5 0 0 1 0 1H5.707l2.147 2.146a.5.5 0 0 1-.708.708l-3-3a.5.5 0 0 1 0-.708l3-3a.5.5 0 1 1 .708.708L5.707 7.5H11.5z"
            />
          </svg>
          <div>
            Viewing 
            <span
              class="sc-dkzDqf gHUvsb"
            >
              9/9
            </span>
          </div>
          <a
            class="sc-hKMtZM eEYTQn"
          >
            Show All
          </a>
        </div>
        <div>
          <input
            class="sc-eCYdqJ gXwLbP"
            type="text"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bczRLJ hWayYc"
    >
      <div
        class="sc-gsnTZi hYgUKR"
      >
        <svg
          fill="currentColor"
          height="1em"
          viewBox="0 0 16 16"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M8 0a8 8 0 1 0 0 16A8 8 0 0 0 8 0zm3.5 7.5a.5.5 0 0 1 0 1H5.707l2.147 2.146a.5.5 0 0 1-.708.708l-3-3a.5.5 0 0 1 0-.708l3-3a.5.5 0 1 1 .708.708L5.707 7.5H11.5z"
          />
        </svg>
        <div>
          Viewing 
          <span
            class="sc-dkzDqf gHUvsb"
          >
            9/9
          </span>
        </div>
        <a
          class="sc-hKMtZM eEYTQn"
        >
          Show All
        </a>
      </div>
      <div>
        <input
          class="sc-eCYdqJ gXwLbP"
          type="text"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
